presto-main/src/main/java/com/facebook/presto/server/GracefulShutdownHandler.java

```diff
@@ -146,6 +155,39 @@ public synchronized void requestShutdown()
                 .collect(toImmutableList());
     }
 
+    private void waitForQueriesToComplete()
+    {
+        //log error if sqlQueryManager is null
```

Comment:
What does this mean?

Reply:
This is not required, let me remove this.